build/gulpfile.js

```diff
@@ -5,12 +5,8 @@ const ngrok = require('ngrok');
 const util = require('gulp-util');
 const nodemon = require('gulp-nodemon');
 
-gulp.task('run', () => {
-  ngrok.connect(3000, (ngrokError, url) => {
```

Comment:
`ngrok@3.x` now uses a Promise-based API instead of callback-based. See: https://github.com/bubenshchykov/ngrok/pull/108/files